pom.xml

```diff
@@ -12,7 +12,7 @@
 		<dependency>
 			<groupId>org.freemarker</groupId>
 			<artifactId>freemarker</artifactId>
-			<version>2.3.17</version>
+			<version>2.3.26-incubating</version>

```

Comment:
should we only use graduated versions?

Reply:
I think ‘incubating’ refers to the fact that freemarker moved to Apache, but the project as a whole is considered ‘incubating’ by them... not that it’s a beta version or anything.